Update full_refresh.md (#8303)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/full_refresh.md

<FAQ path="Seeds/full-refresh-seed" />

## Recommendation
- Set `full_refresh: false` for models of especially large datasets, which you would _never_ want dbt to fully drop and recreate.

> - For existing `full_refresh` configs, you can't override its behavior. If you need different behavior, remove the config or update the config logic using variables so it can change when needed in certain situations.

Reviewer: ```suggestion
- For existing `full_refresh` configs, you can't override their behavior. To change their behavior in certain situations, remove the config or update the config logic using variables.
```
